### PR TITLE
Tune existential deposit and fees to match statemine

### LINKED ISSUE #20: tune existential deposit and fees to match statemine
depends on #21 

* ED like statemine
* transacting KSM should cost the same as on statemine

### PR DESCRIPTION
**Pull request by clangenb**
Closes #20

### CODE REVIEW

File: polkadot-parachains/encointer-runtime/src/lib.rs [hunk 1]
```diff
@@ -227,10 +227,7 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-	pub const ExistentialDeposit: u128 = 1 * MILLIKSM;
-	pub const TransferFee: u128 = 1 * MILLIKSM;
-	pub const CreationFee: u128 = 1 * MILLIKSM;
```
Comment (clangenb) [inline polkadot-parachains/encointer-runtime/src/lib.rs:232]: These were unused and did not exist in statemine either

File: polkadot-parachains/encointer-runtime/src/lib.rs [hunk 2]
```diff
@@ -486,11 +483,6 @@ impl cumulus_pallet_dmp_queue::Config for Runtime {
 }
 
 parameter_types! {
-	pub const AssetDeposit: Balance = 1 * KSM;
-	pub const ApprovalDeposit: Balance = 100 * MILLIKSM;
-	pub const AssetsStringLimit: u32 = 50;
-	pub const MetadataDepositBase: Balance = 1 * KSM;
-	pub const MetadataDepositPerByte: Balance = 10 * MILLIKSM;
```
Comment (clangenb) [inline polkadot-parachains/encointer-runtime/src/lib.rs:493]: These were unused.

File: polkadot-parachains/parachains-common/src/currency.rs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+//! Currency model used in encointer's runtimes.
```
Comment (clangenb) [inline polkadot-parachains/parachains-common/src/currency.rs:1]: I think it is better to use chain-agnostic names, as we can reuse them accross all relay-chains.

However, we need to feature-gate `UNITS` when we start to support polkadot, as the atomic unit on polkadot has been changed to: `UNIT = 10_000_000_000`.

@brenzi better suggestion?

Comment (brenzi) [reply polkadot-parachains/parachains-common/src/currency.rs:1]: sure. units is fine too

**Review by brenzi — APPROVED** [review]

**Pull request merged**